### PR TITLE
🎉 New Source: Holiday API [Low Code CDK]

### LINKED ISSUE #20789: New source: Holiday API
## Tell us about the new connector you’d like to have
Source for the [Holiday API](https://holidayapi.com).

## Describe the context around this new connector
This API is handy for all kinds of business settings, when combined with other internal sources of data. The sync is likely run relatively infrequently, since holidays don't change often. 

This is an API source connector ([docs](https://holidayapi.com/docs)), and likely the "holidays" endpoint is the only relevant one for AirByte.

This service is a freemium one, I'm using it with a free account currently for development, potentially paid if it works out.

## Describe the alternative you are considering or using

Rolling my own using their Python library https://github.com/holidayapi/holidayapi-python

## Are you willing to submit a PR?

Yes.

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Developing new connector for the source: Holiday API*
*Closes #20789*

Resolves:
[🎉 New Source: Holiday API](https://github.com/airbytehq/airbyte/issues/20789)

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## 🚨 User Impact 🚨
- No breaking changes, Just addition of new source
<details><summary><strong>New Connector: Source Holiday API</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Build results

image

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,343 @@
+version: 0.79.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - holidays
+    - countries
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml:7]: You don't have to have multiple streams in check — just one is generally fine, especially for a simple API like this.

File: airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    - languages
+streams:
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-holiday-api/source_holiday_api/manifest.yaml:9]: Schemas are inlined — good, but it strikes me as odd — new versions of Builder should ref out schema definitions into a separate block. Did you use Airbyte OSS? If so, what platform version? Modern (0.57.1 and higher) should work slightly differnetly I think.

**Review by natikgadzhi — COMMENTED** [review]
Overall good, but test failures are valid: 
- formatting
- CATS don't pass: test_read fails because holiday stream is empty, test_check fails. 
- qa checks are failing:
 
```
source-holiday-api - ❌ Failed - Connector must have a language tag in metadata: Expected language tag 'language:python' in the metadata.yaml file, but found 'language:low-code'.
source-holiday-api - ❌ Failed - Python connectors must have a CDK tag in metadata: CDK tag is missing in the metadata file.
source-holiday-api - ❌ Failed - Python connectors must have PyPi publishing enabled: PyPi publishing is not enabled. Please enable it in the metadata.yaml file.
Error: 3 checks failed
```

**Comment by btkcodedev** [comment]
All comments resolved, I was using older version of builder, updated to latest too,
Local CI - All pass
image

**Comment by btkcodedev** [comment]
I'll ping the credentials in slack so that you could update where CI is fetching the credentials from :)

**Comment by btkcodedev** [comment]
All tests are passing, Ready to merge :)

**Comment by natikgadzhi** [comment]
@bnchrch @btkcodedev let's move this one to be a manifest + metadata connector instead!

**Comment by btkcodedev** [comment]
Continued with https://github.com/airbytehq/airbyte/pull/42133

**Pull request closed without merging**